certification/pyxis/submit_test.go

```diff
@@ -49,7 +49,83 @@ var _ = Describe("Pyxis Submit", func() {
 		})
 	})
 
-	Context("updateProject 401 Unauthorized", func() {
+	Context("when a project is submitted with an empty registry", func() {
+		BeforeEach(func() {
```

Comment:
I can't speak to best practices for Ginko, every go project I've worked on just does whatever it wants, with no rhyme or reason, our project is no different with all tests in this package being structured this way. I don't see that Ginko has a setup or a before? Maybe if that existed it would be more valid for our tests.

Reply:
So this was done to be consistent with the rest of the test file?